test

```diff
@@ -47,4 +39,12 @@ if [ -n "${fmtRes}" ]; then
 	exit 255
 fi
 
+# split TEST into an array and prepend REPO_PATH to each local package
+split=(${TEST// / })
+TEST=${split[@]/#/${REPO_PATH}/}
+
+echo "Running tests..."
+go test -i ${TEST}
+sudo -E bash -c "umask 0; PATH=\$GOROOT/bin:\$PATH go test ${COVER} $@ ${TEST}"
```

Comment:
It's for a normal user to be able to clean temporary files. Travis probably doesn't it that as it runs a instance every time, it matters for local runs.